OpenRA.Mods.Common/Activities/Move/MoveAdjacentTo.cs

```diff
@@ -26,14 +26,14 @@ public class MoveAdjacentTo : Activity
 		readonly DomainIndex domainIndex;
 		readonly uint movementClass;
 
-		protected Target target { get; private set; }
+		protected Target Target { get; private set; }
 		protected CPos targetPosition;
 		Activity inner;
 		bool repath;
 
 		public MoveAdjacentTo(Actor self, Target target)
 		{
-			this.target = target;
+			this.Target = target;
```

Comment:
OK.
Generally speaking, is there an easier way than trial and error to spot those cases? Would save both me and reviewers some hassle in the future.